### DISCUSSION TITLE
Self notes (From news update articles when I find it)

### DISCUSSION THREAD

**Discussion by Ongjming** [SmartlyDressedGames/Unturned-3.x-Community#3461] [General]
Regards to placing barricades (Beds and Sentries) on vehicles:
[3.30.2.0 Update Notes](https://steamcommunity.com/ogg/304930/announcements/detail/1602632162280022511)
Allow_Placement_On_Vehicle (True/False) To be placed inside .dat file of the barricades in question.
False by default for Beds and Sentries, all other barricades are True by default.

[3.20.5.0 Update Notes](https://steamcommunity.com/ogg/304930/announcements/detail/3460398821839264088)
Bypass_Buildable_Mobility (True/False) To be placed inside Config.json of the Map in question.
With this toggled to True, barricades which are Sentries and Beds can be placed on vehicles so long it is within that map, overrides individual "Allow_Placement_On_Vehicle" setting.

Supports_Mobile_Buildables (I have no idea which news update post this appears in, I can't find it anywhere except in .dat of Train)
Based on what was written in there, I assumed it allowed Beds and Sentries to be placed on that particular vehicle this line was in.
Probably defaults to True when the line is present, and defaults to false when it is not present.

**Comment by Ongjming** [comment]
Regards to Camera settings for vehicles:
Cam_Passenger_Offset : This value refers to the height offset (Y coordinate increase) of the person's view while in the seat's position, includes every single person in the vehicle and overrides "Cam_Driver_Offset"
Cam_Driver_Offset: This value increases the height offset for just the driver.

In Unity:
If your vehicle has a seat_x position that also has a turret appointed to it, whatever position the seat_x child of the turret parent will supersede the seat_x position that is parented to seat parent.

To check roughly if your character can see anything, take the charactermeshreplacementtest thingy from the examplepackage and stick it to the Seat_x position. The "F" on the model's face is roughly where their eyes (View) should be at. (So at this point you can either offset it within unity or offset it using the above lines without having to remasterbundle. Remember to remove that thingy after.)

-------------------------------------------------------------------------------------------------------------------------------------------
Cam_Follow_Distance: It is a value that the camera revolve around the seat_x point, like an orb. (In unity can roughly see the distance by attaching a light source as a child to the seat_x, Range value to control the orb's size, this is the boundary you should zoom the unity camera view to, in order to see how it would appear in game. Once satisfied, the range value can be copy pasted as the Cam_Follow Distance. Then remember to remove the light source+child)

**Comment by Ongjming** [comment]
Regards to explosion on vehicle:
[3.29.5.0 Update Notes](https://steamcommunity.com/ogg/304930/announcements/detail/1773763876479209963)
Child_Explosion_Armor_Multiplier: Value from 0 to 1 and maybe more, idk. When placed in a vehicle's .dat file, Barricades that are build on the vehicle, when they get hit with explosion damage, the barricades will get 100% of the damage and the vehicle itself will also get 100% dmg Multiplied by this value. (e.g if this value is 0.5 and the explosive does 500 damage, the barricade receives 500 dmg, the vehicle will also receive 250 dmg.) Default value of this currently appears to be 0.2 according to the news post.

Passenger_Explosion_Armor: Values also from 0 to 1, maybe more too idk. From discord and websites, it appears that this setting modifies the dmg the vehicle's passengers take when the vehicle gets hit by explosion dmg. However, I am not really sure if passengers will still get damaged if the physical explosion only hits the vehicle without hitting the part where the passenger is at.

**Comment by Ongjming** [comment]
[3.23.7.0 Update Notes](https://steamcommunity.com/ogg/304930/announcements/detail/3681175298249362221)
Old Ballistic_Drop values would be converted automatically by the game into relevant Bullet_Gravity_Multiplier values via some calculation.
Recoil_Aim is gone (Recoil multiplier for aiming down sights, gone = no replacement)
SpotLight_Enabled: Not really sure what this does, maybe it allows for unity placed spotlights to be lit instead of the default lights? (i.e. we can finally have different colored/styles of light?)
Allow_Lobby_Groups server config option: I have no clue what this does either, maybe it is kinda like a waiting room thingy where people can be in queue to load files and chat at the same time?
(Will edit once I find out more)

**Reply by Jdance-Media** [reply]
![image](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/67397422/9f7caf6d-fae9-467a-a0a3-26043366aeb1)
The entire community forgot what lobbies are. 🤦‍♂️ Had many people ask around what they were after this update.
